draft-ietf-ppm-dap.md

```diff
@@ -2653,59 +2638,57 @@ running aggregation jobs ({{aggregate-flow}}) until the Collector initiates a
 collection job. This is because, in general, the aggregation parameter is not
 known until this point. In certain situations it is possible to predict the
 aggregation parameter in advance. For example, for Prio3 the only valid
-aggregation parameter is the empty string. For these reasons, the collection job
-is handled asynchronously. If aggregation is performed eagerly, the Leader MUST
-validate that the aggregation parameter received in the `CollectionJobReq`
-matches the aggregation parameter used in aggregations.
+aggregation parameter is the empty string.
 
 Upon receipt of a `CollectionJobReq`, the Leader begins by checking that it
-recognizes the task ID in the request path. If not, it MUST abort with error
-`unrecognizedTask`.
+recognizes the task ID. If not, it MUST abort with error `unrecognizedTask`.
+
+Next, the Leader checks that the indicated batch mode matches the task's batch
+mode. If not, then the Leader MUST abort with error `invalidMessage`.
 
 Next, the Leader checks that the aggregation parameter is valid as described in
-{{agg-param-validation}}. If the aggregation parameter is invalid, then the
-Leader MUST abort with error `invalidAggregationParameter`.
+{{agg-param-validation}}. If not, then the Leader MUST abort with error
+`invalidAggregationParameter`.
+
+If aggregation is performed eagerly, the Leader MUST validate that the
+aggregation parameter received in the `CollectionJobReq` matches the aggregation
+parameter used in aggregations.
+
+Changing a collection job's parameters is illegal, so further PUT requests to
+the collection job with a different `CollectionJobReq` MUST fail.
```

Comment:
Let's replace "MUST fail" with a specific action, like an abort.

Reply:
Sure. I made a similar change to the equivalent text in the aggregation job init section.